app/features/notes_generator/tests/test_core.py

```diff
@@ -0,0 +1,78 @@
+from app.features.notes_generator.core import executor
```

Comment:
The testing structure must follow the same principle and structure from the other tools.